src/mailadm/conn.py

```diff
@@ -232,6 +232,48 @@ def get_expired_users(self, sysdate):
             users.append(UserInfo(*args))
         return users
 
+    def get_users_to_warn(self, sysdate: int) -> [{}]:
+        allusers = self.get_user_list()
+        users_to_warn = []
+        for user in allusers:
+            if user.token_name == "created in mailcow":
+                continue
+            year = 31536000
+            month = 2592000
+            week = 604800
+            day = 86400
+            warnmsg = "Your account will expire in ?. You should look for an alternative email " \
+                "provider right now.\nWith Delta Chat, you can keep all your chats and " \
+                "conversations. Just use the AEAP mechanism to tell your contacts of your " \
+                "address migration: https://delta.chat/en/2022-09-14-aeap\n\nYour %s team" % \
+                (self.config.mail_domain,)
```

Comment:
Let's just say "Your account will expire in ?" here for now.  The other (AEAP) part are not good enough and might be quite out of context for the user.

Reply:
also please use a "format" string, like "Warning! Your account will expire in {}." You can then call "message.format(warning_text)". This way you don't need to use "replace" which is brittle.